src/shared/services/HttpService.ts

```diff
@@ -61,8 +61,10 @@ class WrappedLemmyHttpClient {
               state: !(res === undefined || res === null) ? "success" : "empty",
             };
           } catch (error) {
-            console.error(`API error: ${error}`);
-            toast(I18NextService.i18n.t(error), "danger");
+            if (!silent) {
+              console.error(`API error: ${error}`);
+              toast(I18NextService.i18n.t(error), "danger");
```

Comment:
The user will see a toast that says "danger"? Could this be less scary or somewhat informative?